sympy/ntheory/digits.py

```diff
@@ -6,7 +6,7 @@
 from sympy.utilities.iterables import multiset, is_palindromic as _palindromic
 
 
-def digits(n, b=10):
+def digits(n, b=10, bits=None):
     """
     Return a list of the digits of ``n`` in base ``b``. The first
     element in the list is ``b`` (or ``-b`` if ``n`` is negative).
```

Comment:
Can you please add the missing docstrings with `Parameters` section giving a description of `n, b,` and `bits`?. Also, please add examples in `Examples` of using `bits` when it is not `None`. Similarly, also add tests for `bits` not equal to `None`.

Reply:
Okay. Working on it.